### PR TITLE
UX-974/Track untracked actions and leave Bugsnag breadcrumbs for each path change and each action

### PR DESCRIPTION
**Pull request by knnguy**
Changes made:
- Tracked untracked actions for Segment
- Added breadcrumbs with type 'navigation' for each UI path change. These correspond with the tracked pages in Segment. 
- Added breadcrumbs for each action, such as creating clusters, deleting node, scaling masters, etc.
- Added customer tier and du version into the App tab in Bugsnag

Screenshots taken from Bugsnag

Screen Shot 2021-05-20 at 11 45 57 AM

Screen Shot 2021-05-20 at 2 54 17 PM

### CODE REVIEW

**Comment by knnguy** [comment]
I'm assuming that this is the DU version in features.json. Is this correct? How about for DDUs?
Screen Shot 2021-05-20 at 1 11 46 PM

File: src/app/core/containers/AppContainer.tsx [hunk 1]
```diff
@@ -143,7 +145,12 @@ const AppContainer = () => {
 
   useEffect(() => {
     const unlisten = history.listen((location) => {
-      trackPage(`${location.pathname}${location.hash}`)
+      const url = `${location.pathname}${location.hash}`
```
Comment (jruddell) [inline src/app/core/containers/AppContainer.tsx:148]: wouldnt the getCurrentRoute on the Route class do the same thing?

**Review by jruddell — APPROVED** [review]
Really awesome stuff here, great work!

**Comment by knnguy** [comment]
I made a small change where I moved `Bugsnag.addBreadcrumb` for each route into the `trackPage` function so that we can just call `Route.find` once and track the route for both Segment and Bugsnag at the same time

**Review by jruddell — APPROVED** [review]

**Pull request merged**